packages/flutter_markdown/pubspec.yaml

```diff
@@ -0,0 +1,11 @@
+name: flutter_markdown
+description: A markdown renderer for Flutter.
+version: 0.1.0
+author: Flutter Authors <flutter-dev@googlegroups.com>
+homepage: http://flutter.io
+
+dependencies:
+  flutter:
+    path: ../flutter
+  markdown: "0.9.0"
+  string_scanner: "0.1.4+1"
```

Comment:
to add tests, add the following to the pubspec.yaml:
```
dev_dependencies:
  flutter_tools:
    path: ../flutter_tools
  test: any # constrained by the dependency in flutter_tools
```
...and add flutter_markdown to `//travis/test.sh` in the same way that flutter_sprites is there.
Then, add tests the same way the flutter package does:
https://github.com/flutter/flutter/tree/master/packages/flutter/test

You'll want to test things like:
* do you get the right tree structure for a given input, for a wide variety of inputs.
* does the input survive unchanged (i.e. object identities are not affected) when you change the dimensions of the device (e.g. rotating it). In particular, ensuring that nothing gets reparsed.
* do lists get the right numbering when there's multiple lists in a row.
* anything else you think people might break when they later change the code.

Please do this in this patch, so that the analyzer has analyzed the patch (the analyzer only checks code that has tests).